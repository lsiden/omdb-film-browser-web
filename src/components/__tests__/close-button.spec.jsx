import React from 'react'
import TestRenderer from 'react-test-renderer';
import { StaticRouter } from 'react-router'

import CloseButton from 'components/close-button'

const createWrapper = () => TestRenderer.create(
  <StaticRouter>
    <CloseButton onClick={jest.fn()} />
  </StaticRouter>,
)

test('CloseButton', () => {
  expect(createWrapper()).toMatchSnapshot()
})
