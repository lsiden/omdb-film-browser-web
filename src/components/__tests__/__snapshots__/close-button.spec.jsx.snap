// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CloseButton 1`] = `
<button
  className="btn btn-link"
  onClick={[Function]}
  style={
    Object {
      "fontSize": 20,
      "margin": 20,
    }
  }
  type="button"
>
  âœ•
</button>
`;
